### ISSUE TITLE
New contributor self intro [Khoo Yong Jie]

### ISSUE THREAD

**Issue by YongJieYongJie** [TEAMMATES/teammates#4937]
*inserts token... here comes a new contributor!*

Hello everyone!
I'm **Yong Jie**, currently taking a minor in CS at NUS.
I hope to participate in TEAMMATES as part of GSoC.

I took [CS2103][1] last semester and is taking [CS3226][2] now.
I have always been interested in coding, and have written stuff that ranged from useful to random and perhaps even semi-legal (i.e., bots).

I've used **Java** for basic networking programming in CS2105 (Computer Networking) and for information retrieval in CS2108 (Media Computing) last semester. Currently, I'm using **LAMP** with **AngularJS** for CS3226 project, and is writing a personal web scraper in **Ruby**.

I deeply regrets not taking CS as my major =(
But in any case, I'm making full use of the CS minor to experience programming in a group and is taking my third project-based module this semester (out of maximum of six CS modules I'm allowed).

**Here are the required information**:
[link to my app][3]
[link to my fork][4]
[1]: http://www.comp.nus.edu.sg/~cs2103/AY1516S2/ "Software Engineering"
[2]: www.comp.nus.edu.sg/~stevenha/cs3226.html "Web Programming and Applications" 
[3]: https://teammates-yongjie.appspot.com/ "Yong Jie's fork"
[4]: https://github.com/YongJieYongJie/teammates "Yong Jie's app"

Screenshot of UI Change / Improvement:
![image](https://cloud.githubusercontent.com/assets/14101781/13603848/10ea19b2-e57b-11e5-9299-3ac0b16222eb.png)
Basically, I corrected uneven padding of "Student Login" button, and added content border for improved visual focus (inspired by Material Design)

**Comment by junhaoyap** [comment]
Welcome to the TEAMMATES contributor community @YongJieYongJie, good work on moving into Phase D and following the instructions!

Please refer to our 'Instructions for Google Summer of Code Applicants' for guidance on how you can move forward in applying to contribute to TEAMMATES under GSoC :)

It'll also be great if you can pick some issues and start contributing!
